### ISSUE TITLE
Using multiple cytoscape instances leads to cy instance mismatch

### ISSUE THREAD

**Issue by gianluca-pepe** [dmx-systems/cytoscape-edge-connections#7]
Hi,
I have different graphs built with cytoscape.js and on each of them I initialise the edge connection plugin.
Something like this:
```js
const cy1 = cytoscape()
const cy2 = cytoscape()

const cy1Ec = cy1.edgeConnections()
cy1Ec.addEdges(...)

const cy2Ec = cy2.edgeConnections()
cy2Ec.addEdges(...)
// ...sometime later
cy1.style(...) // this fires the event handler set by cy1Ec but internally it uses cy2 causing the data inconsistency exception
```

When an event handler set by cy1Ec gets triggered, it uses the last cytoscape instance on which `edgeConnections()` has been called (cy2) instead of the correct one (cy1). 
I haven't tried yet, but I guess also adding a new edge on cy1 after `cy2.edgeConnections()` would result in the addition of the edge on cy2 instance, in this case I could easily avoid the issue calling again `cy1.edgeConnections()` but I can't control when the event handler will be triggered.
Am I missing something?

**Comment by jri** [comment]
Hi,
yes, I see the problem. At the moment the edge-connections plugin does not support your case, that is multiple cytoscape instances. I can fix the plugin soon. Please excuse the late reply!
Thank you for reporting!

**Comment by jri** [comment]
This is now fixed. The plugin should now work with multiple Cytoscape instances.

As long as there is no new plugin release you can try it by putting the repo URL into your dependencies (instead the version number):
```
  "dependencies": {
    "cytoscape-edge-connections": "https://github.com/dmx-systems/cytoscape-edge-connections.git"
  }
```
If it works for you I'll release it as 0.5.

**Comment by gianluca-pepe** [comment]
Thank you for the fix!
I cannot install it through repo URL due to corporate policies about SSL certificates, having it working via npm would be ideal, even with a beta release.

**Comment by jri** [comment]
OK. The plugin is now released as `0.5.0`.

**Comment by gianluca-pepe** [comment]
Great, thank you!
